at_client/lib/src/manager/at_client_manager.dart

```diff
@@ -30,24 +30,28 @@ class AtClientManager {
 
   AtClientManager._internal();
 
+  @deprecated
   factory AtClientManager.getInstance() {
```

Comment:
@gkc I am not fully able to remove the singleton related artifacts in order to maintain backward compatibility. Within the at client SDK we will be removing the references to AtClientManager.getInstance(). In widgets and app code, there will be references to AtClientManager.getInstance() which will return the static object _singleton. I am skeptical whether this may cause any bugs (having constructed instance of AtClientManager and _singleton)

Reply:
I made a change this morning. Instead of creating new instance of AtClientManager inside setCurrentAtSign, I passed "this" to AtClientImpl.create and SyncServiceImpl.create..So callers using getInstance the old way will have the singleton reference of AtClientManager..callers using the new constructor AtClientManager(_atSign)  will have one manager per atsign. Is this fine?

Reply:
Yes I think that's fine, given the objectives at this point. There's nothing fundamentally wrong with how this works from a client usage perspective; at this point this change is aimed at making the AtClient, SyncService and NotificationService implementations more testable